docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
So, https://github.com/rails-api/active_model_serializers/pull/1294 was merged, does that address this issue or should we revert it and continue here?